PicoVR Unity SDK/chapter_seven.txt

```diff
@@ -1,281 +1,602 @@
 
-.. image:: _static/7.3.png
+.. image:: _static/7.1a.png
 
-Figure 7.3 Fade Time and Fade Color after Opening the Scenes
+Figure 7.1 Selecting a Platform to Publish the App to.
 
-7.3 	Limited frame rate
--------------------------------------
+6.	Complete the app-related information for the corresponding platform:
 
-The SDK provides the function of limiting frame rate, which is opened by unchecking the Use Default FPS option as follows
+.. image:: _static/8.2.png
 
-.. image:: _static/7.4.png
+Figure 7.2 Entering the App-related Information
 
-Figure 7.4 Opening of limited frame rate
+Enter the values for each field carefully because the app type cannot be modified once completed. If there exists an inside-props payment in your application, you must use the developer backend to add a merchandise code for unified management.
 
-Then enter the desired frame rate value in the FPS text box.
+7.1.1 	Description of Payments
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-7.4 	Configure the size of EyeBuffer
----------------------------------------------
+Pico currently has two types of payment methods: *P currency payment (application type)*, and *commodity code payment (game type)*. An application can only use one of the two payment methods.
+After the app has been created successfully, the developer platform will assign it a number of strings including an ``APPID``, ``APP KEY``, ``APP Secret``, and a developer ID:
 
-The SDK provides the function of configuring the size of the Eyebuffer, which is opened by unchecking the Use Default RenderTexture option as follows
+.. image:: _static/8.3.png
 
-.. image:: _static/7.5.png
+Figure 7.3 APP ID, APP KEY, APP Secret
 
-Figure 7.5 Configuration size of Eyebuffer
+Once these strings have been provided, select In-Game Payment Settings to configure in-game purchase information:
 
-7.5 	Custom Application Launch Animation
----------------------------------------------
+.. image:: _static/7.3a.png
 
-The SDK provides the function of customizing the application to launch animation. Click the menu bar Pvr_UnitySDK/ Splash Screen to open the Splash Screen editing interface, as shown in the figure:
+Figure 7.4 Configuring in-game purchase information
 
-.. image:: _static/7.6.png
+Merchandise codes have the following attributes:
 
-Figure 7.6 Splash Screen option
+*	The first character is a letter.
+*	Only letter and number symbols are allowed.
+*	The code can have up to 20 symbols.
+*	Codes for different props cannot be duplicated. 
 
-SplashScreenType can be set to the following values corresponding to a different application startup animation:
+The props are categorized as *consumable* props and *inconsumable* props. Consumable props consist of merchandise that can be purchased repeatedly, such as gold coins, blood bottles, etc. Inconsumable props are one-time purchases such as weapons, checkpoint unlocks, etc.
```

Comment:
Remove this line because in-game purchase is not provided in [us-dev.picovr.com](url)
...end here.